Cargo.toml

```diff
@@ -71,40 +71,40 @@ path = "system-parachains/asset-hubs/asset-hub-polkadot"
 
 [workspace.dependencies.asset-test-utils]
 version = "*"
-git = "https://github.com/paritytech/polkadot-sdk.git"
+git = "https://github.com/paritytech/polkadot-sdk"
 branch = "oty-2503-paseo"
```

Comment:
why specific branch? why not commit from master for example? are we starting a specific version of sdk for the migration?